src/libs/Geometry/geom_static.cpp

```diff
@@ -19,20 +42,9 @@ GEOS *CreateGeometry(const char *fname, const char *lightname, GEOM_SERVICE &srv
 // geometry constructor does all init
 GEOM::GEOM(const char *fname, const char *lightname, GEOM_SERVICE &_srv, long flags) : srv(_srv)
 {
-    unsigned int *colData = nullptr;
-    auto flsz = 0;
-    if (lightname != nullptr)
-    {
-        auto ltfl = srv.OpenFile(lightname);
-        flsz = srv.FileSize(lightname);
-        if (flsz > 0)
-        {
-            colData = static_cast<unsigned *>(srv.malloc(flsz));
-            srv.ReadFile(ltfl, colData, flsz);
-        }
-        else
-            lightname = nullptr;
-        srv.CloseFile(ltfl);
+    std::vector<unsigned int> colData;
```

Comment:
please, use uint32_t (and add `#include <cstdint>`)
I see in the original code it was `unsigned int`, but let's fix that up when we go through